pkgs/top-level/python-packages.nix

```diff
@@ -25665,21 +25657,18 @@ EOF
 
     src = pkgs.fetchurl rec {
       url = "mirror://pypi/g/graphite-web/${name}.tar.gz";
-      sha256 = "1c0kclbv8shv9nvjx19wqm4asia58s3qmd9fapchc6y9fjpjax6q";
+      sha256 = "0q8bwlj75jqyzmazfsi5sa26xl58ssa8wdxm2l4j0jqyn8xpfnmc";
     };
 
-    propagatedBuildInputs = with self; [ django django_tagging_0_3 whisper pycairo ldap memcached pytz ];
+    propagatedBuildInputs = with self; [ django_1_8 (django_tagging.override { django = django_1_8; }) whisper pycairo ldap memcached pytz urllib3 cairocffi scandir pyparsing ];
```

Comment:
No overriding within the package set. All Django packages will need to depend just on `django`, and not on a specific version. If you application needs a specific version, then you need to override the whole Python package set when deploying it.